### DISCUSSION TITLE
Some confusions about topsapp.py

### DISCUSSION THREAD

**Discussion by A1b8** [isce-framework/isce2#629] [Q&A] [answered]
Hello everyone,
I ran the topsApp.py to get coseismic deformation of an earthquakes. But I do not know whether the planar trend has been removed or not. And what conditions can I apply the “unwrap 2 stage"?  
I would appreciate you solving my confusion.
Thanks!

**Comment by fdelgadodelapuente** [comment, marked as answer]
Hi A1b8,

The "planar trend" you mean is in the general case a ramp that can be linear, bilinear, quadratic or something more complex if you have lots of ionospheric delays. Usually linear and bilinear trends are fine to account for these signals. ISCE does not remove them from the data, but you can remove them easily with grdtrend in GMT.

Francisco

**Reply by A1b8** [reply]
Thanks for your detailed answer!
